edbdeploy/project.py

```diff
@@ -636,7 +636,8 @@ def destroy(self):
     def deploy(self, no_install_collection,
                pre_deploy_ansible=None,
                post_deploy_ansible=None,
-               skip_main_playbook=False):
+               skip_main_playbook=False,
+               enable_pipelining=False):
```

Comment:
Let's keep default to be true. This way, out of the box, we will be using the performant option, and a user will have the choice to disable it.